test/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ endif()
 
 if (MIOPEN_TEST_GFX90a)
     if(WORKAROUND_ISSUE_1093)
-        list(APPEND SKIP_TESTS test_find_db test_main test_immed_conv2d)
+        list(APPEND SKIP_TESTS test_conv2d test_find_db test_main test_immed_conv2d)
```

Comment:
Hi @shaojiewang , by reviewing "all changed", this test_conv2d is still not removed from the SKIP_TESTS. Is that an error? Thanks!

Reply:
Yes. I guess that at least test_conv2d should be removed from W/A for #1093 in `test/CMakeLists.txt`.

Even more, I suspect that when all the gfx90a assembly errors fixed (or avoided via W/A), the whole `WORKAROUND_ISSUE_1093` in `test/CMakeLists.txt` will not be necessary anymore. This can be easily checked by passing `-DWORKAROUND_ISSUE_1093=Off` to CMake.

@shaojiewang Please make neccesary changes of `test/CMakeLists.txt` right in this PR, thanks.

Reply:
changed